packages/mobile_app/lib/common/triangle_painter.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+import 'package:my_prophet/theme/app_colors.dart';
+
+class TrianglePainter extends CustomPainter {
```

Comment:
Would be grateful if you could add a comment on what this class is used for. 
My guess that this is a triangle that is presented at the right from the prediction numbers, but this should be stated somewhere.

Reply:
Yes, it is a triangle presented near prediction numbers, but can be used otherwise, just needs a bool if it is negative.

Reply:
I mean, would be nice if you would add a header of comments with small description of what this thing does